optimize/self-managed/optimize-deployment/migration-update/3.11_8.3-to-3.12_8.4.md

```diff
@@ -0,0 +1,119 @@
+---
+id: 3.11_8.3-to-3.12_8.4
+title: "Update notes (8.3/3.11 to 8.4/3.12)"
+---
+
+:::note Heads up!
+To update Optimize to version 8.4/3.12, perform the steps in the [migration and update instructions](./instructions.md).
+:::
+
+The update to 8.4/3.12 can be performed from any 8.3/3.11 release.
+
+Here you will find information about:
+
+- Limitations
+- Known issues
+- Changes in supported environments
+- Changes in behavior (for example, due to a new feature)
+- Changes in translation resources
+
+## Changes in supported environments
+
+### Elasticsearch
+
+With 8.4/3.12, Optimize now supports Elasticsearch `8.9`.
+Additionally, please note there are temporary changes in Optimize's Elasticsearch support as detailed below:
+
+| Optimize version                        | Elasticsearch version            |
+| --------------------------------------- | -------------------------------- |
+| Optimize 3.10.0 - Optimize 3.10.3       | 7.16.2+, 7.17.0+, 8.5.0+, 8.6.0+ |
+| Optimize 3.10.4                         | 7.16.2+, 7.17.0+, 8.7.0+, 8.8.0+ |
+| Optimize 3.10.5 - Optimize 8.3.x/3.11.x | 7.16.2+, 7.17.0+, 8.5.0+, 8.6.0+ |
+| Optimize 8.3.x/3.11.x                   | 8.8.0+                           |
+| Optimize 8.4.x/3.12.x                   | 8.9.0+                           |
+
+See the [supported environments]($docs$/reference/supported-environments) section for the full range of supported versions.
+
+If you need to update your Elasticsearch cluster, refer to the general [Elasticsearch update guide](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html). Usually, the only thing you need to do is perform a [rolling update](https://www.elastic.co/guide/en/elasticsearch/reference/current/rolling-upgrades.html).
```

Comment:
I worry about using a term like "usually" -- if there is another instance where I might need to do something else, please clarify 👍 If a rolling update is all I will need 99% of the time, we can remove the "usually" here.

Reply:
On major version changes, rolling upgrades are often not supported by ES. But it's very much out of our hands. Happy to change to whatever you think, please advise as to what you think is best fit 🙂